src/main/java/engine/movegen/KnightMovesGen.java

```diff
@@ -24,6 +24,5 @@ public List<Move> generate (Board board, Piece knight) {
         }
         super.removeInvalid(this.moves, board);
         return this.moves;
-
```

Comment:
whoops sorry don't know why this is here, didn't mean to commit (don't even remember opening the file). But I don't think it's an issue anyways so whatever.